### PR TITLE
Remove old references to CUSTOMDUMPSOFTWAREVERSIONS

### PR DESCRIPTION
**Pull request by jfy133**
Removes left over CUSTOMDUMPSOFTWAREVERSIONS references from template that has been since replaced with a dedicated special function

## PR checklist

- [x] This comment contains a description of changes (with reason)
- [x] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated

### CODE REVIEW

**Review by matthdsm — APPROVED** [review]

**Review by ewels — CHANGES_REQUESTED** [review]
Can we also add a lint test for this please?

**Comment by jfy133** [comment]
@mirpedrol @mashehu help!

**Comment by mashehu** [comment]
haha, you ran into the same nf-core tools trap as @Joon-Klaps did 🙂 
Screenshot 2024-03-26 at 13 15 53

**Comment by mashehu** [comment]
but I can have a look after I found a fix for #2890

**Comment by ewels** [comment]
image

File: nf_core/lint/configs.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        fn = Path(self.wf_path, file_path)
+        passed: List[str] = []
+        failed: List[str] = []
+        ignored: List[str] = []
+
+        ignore_configs = self.lint_config.get(lint_name, [])
+
+        # Return a failed status if we can't find the file
+        if not fn.is_file():
+            return {"failed": [f"`${file_path}` not found"]}
```
[the remark this answers is not in the document]
Comment (mashehu) [reply nf_core/lint/configs.py:22]: better to have them ignore it, imo

File: nf_core/lint/configs.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+        # check if withName sections are present in config, but not in workflow files
+        for section in sections:
+            if section not in ignore_configs:
+                if not any(section in nf_file.read_text() for nf_file in nf_files):
+                    failed.append(
+                        f"`{file_path}` contains `withName:{section}`, but the corresponding process is not present in any of the following workflow files: `{nf_files}`."
+                    )
+                else:
+                    passed.append(f"both `{file_path}` and `{[str(f) for f in nf_files]} contain `{section}`.")
```
[the remark this answers is not in the document]
Comment (mashehu) [reply nf_core/lint/configs.py:44]: okay, but I'll add a debuggin log, so it is a bit easier to debug

**Review by mirpedrol — APPROVED** [review]
LGTM!

**Pull request merged**